### ISSUE TITLE
About Bromite crashes Bromite

### ISSUE THREAD

**Issue by Ajaja** [bromite/bromite#710]
### Bromite version

Version: `85.0.4183.82`
Arch: `arm64`

### Is this bug about the SystemWebView?
No

### Is the bug reproducible with latest version?

Yes

### Can the bug be reproduced with corresponding Chromium version?
No

### Is the bug a crash?

Yes

### Describe the bug

"About Bromite" crashes Bromite

### Steps to reproduce the bug

Steps to reproduce the bug:
1. Go to 'Settings'
2. Click on 'About Bromite'
3. See error

**Comment by csagan5** [comment]
@Ajaja can you get the crash through `adb logcat`?

**Comment by Ajaja** [comment]
@csagan5 
No, I can't
But I get the crash on all my 3 devices: Android 5 (arm), Android 8 (arm64) and Android 8 (arm64). 
So, it's easy to reproduce, I think.

**Comment by Ajaja** [comment]
Logcat Reader shows this:
```FATAL EXCEPTION: main
Process: org.bromite.bromite, PID: 23707
java.lang.RuntimeException: Unable to start activity ComponentInfo{org.bromite.bromite/org.chromium.chrome.browser.settings.SettingsActivity}: android.view.InflateException: Binary XML file line #14: Error inflating class (not found)org.chromium.chrome.browser.settings.HyperlinkPreference
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2521)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2601)
	at android.app.ActivityThread.access$800(ActivityThread.java:178)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1470)
	at android.os.Handler.dispatchMessage(Handler.java:111)
	at android.os.Looper.loop(Looper.java:194)
	at android.app.ActivityThread.main(ActivityThread.java:5637)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:960)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)
Caused by: android.view.InflateException: Binary XML file line #14: Error inflating class (not found)org.chromium.chrome.browser.settings.HyperlinkPreference
	at hg.b(chromium-ChromeModernPublic.apk-stable-1598054901:9)
	at hg.d(chromium-ChromeModernPublic.apk-stable-1598054901:23)
	at hg.c(chromium-ChromeModernPublic.apk-stable-1598054901:8)
	at Zf.b1(chromium-ChromeModernPublic.apk-stable-1598054901:8)
	at Dh1.a(chromium-ChromeModernPublic.apk-stable-1598054901:2)
	at org.chromium.chrome.browser.about_settings.AboutChromeSettings.d1(chromium-ChromeModernPublic.apk-stable-1598054901:2)
	at Zf.h0(chromium-ChromeModernPublic.apk-stable-1598054901:10)
	at mb.U(chromium-ChromeModernPublic.apk-stable-1598054901:73)
	at Gb.b(chromium-ChromeModernPublic.apk-stable-1598054901:24)
	at Gb.o(chromium-ChromeModernPublic.apk-stable-1598054901:11)
	at mb.C(chromium-ChromeModernPublic.apk-stable-1598054901:45)
	at mb.a0(chromium-ChromeModernPublic.apk-stable-1598054901:10)
	at mb.B(chromium-ChromeModernPublic.apk-stable-1598054901:14)
	at mb.v(chromium-ChromeModernPublic.apk-stable-1598054901:5)
	at Na.onStart(chromium-ChromeModernPublic.apk-stable-1598054901:9)
	at Y0.onStart(chromium-ChromeModernPublic.apk-stable-1598054901:1)
	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1244)
	at android.app.Activity.performStart(Activity.java:6128)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2478)
	... 10 more
Caused by: java.lang.ClassNotFoundException: org.chromium.chrome.browser.settings.HyperlinkPreference
	at java.lang.Class.classForName(Native Method)
	at java.lang.Class.forName(Class.java:309)
	at hg.a(chromium-ChromeModernPublic.apk-stable-1598054901:9)
	at hg.b(chromium-ChromeModernPublic.apk-stable-1598054901:3)
	... 28 more
Caused by: java.lang.ClassNotFoundException: Didn't find class "org.chromium.chrome.browser.settings.HyperlinkPreference" on path: DexPathList[[zip file "/data/app/org.bromite.bromite-1/base.apk"],nativeLibraryDirectories=[/data/app/org.bromite.bromite-1/lib/arm, /vendor/lib, /system/lib]]
	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:511)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:469)
	... 32 more
	Suppressed: java.lang.ClassNotFoundException: org.chromium.chrome.browser.settings.HyperlinkPreference
		at java.lang.Class.classForName(Native Method)
		at java.lang.BootClassLoader.findClass(ClassLoader.java:781)
		at java.lang.BootClassLoader.loadClass(ClassLoader.java:841)
		at java.lang.ClassLoader.loadClass(ClassLoader.java:504)
		... 33 more
	Caused by: java.lang.NoClassDefFoundError: Class not found using the boot class loader; no stack available```

**Comment by impreza233** [comment]
I can reproduce this too, on Android 10. Same logcat as before.

**Comment by shivangchopra22** [comment]
I can reproduce this too.

**Comment by csagan5** [comment]
Alright, thanks; will be fixed on next release.

**Comment by chromer030** [comment]
I can reproduce this too, on Android 7.1.2 , arm64.

**Comment by csagan5** [comment]
Fixed in `85.0.4183.84`.

**Comment by whour** [comment]
> Fixed in `85.0.4183.84`.

This version and the previous version `85.0.4183.82` keep crashing when I open.

**Comment by shivangchopra22** [comment]
Facing the same bug as @whour on both the versions, so downgraded to 84.0.4147.132

**Comment by csagan5** [comment]
@whour @shivangchopra22 that is a different issue, see #565

**Comment by chromer030** [comment]
confirm that issue resolved in 85.0.4183.84 , android 7.1.2 arm64
